packages/client/test/integration/mocks/mockchain.ts

```diff
@@ -23,16 +24,22 @@ export default class MockChain extends Chain {
 
   async build() {
     const blocks: Block[] = []
+    const mergeBlockNumber = this.config.chainCommon.hardforkBlockBN('merge')?.toNumber();
     for (let number = 0; number < this.height; number++) {
       const block = Block.fromBlockData({
         header: {
           number: number + 1,
-          difficulty: 1,
+          difficulty: number > 10 ? 0 : new BN(1),
           parentHash: number ? blocks[number - 1].hash() : (this.genesis as any).hash,
         },
       })
       blocks.push(block)
     }
-    await this.putBlocks(blocks)
+    try {
+      await this.putBlocks(blocks)
+    }
+    catch (err) {
+      console.log(err);
```

Comment:
wouldn't this end up printing the error to console anyway even without the added code if it throws?

Reply:
Yes, just bad console logging on my part trying to figure out that EIP1559 error I kept seeing.